sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -404,10 +406,11 @@ object RapidsConf {
       "\"ASYNC\", and \"NONE\". With \"DEFAULT\", the RMM pool allocator is used; with " +
       "\"ARENA\", the RMM arena allocator is used; with \"ASYNC\", the new CUDA stream-ordered " +
       "memory allocator in CUDA 11.2+ is used. If set to \"NONE\", pooling is disabled and RMM " +
-      "just passes through to CUDA memory allocation directly. Note: \"ARENA\" is the " +
-      "recommended pool allocator if CUDF is built with Per-Thread Default Stream (PTDS).")
+      "just passes through to CUDA memory allocation directly. When unset, if the CUDA driver is " +
+      "newer than 11.2, the \"ASYNC\" allocator is used; otherwise, the \"ARENA\" allocator is " +
+      "used.")
     .stringConf
-    .createWithDefault("ARENA")
+    .createWithDefault("")
```

Comment:
Given the number of users on CUDA < 11.2 will be dwindling over time quickly, wondering if we should just have `ASYNC` automatically fallback to `ARENA` with a log message on startup when the fallback occurs.

Reply:
Done.